openedx/core/djangoapps/organization_api/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""
+Organization API.
+
+WARNING: This API is intended to move into the 'edx-organizations' repo.
```

Comment:
After discussion it was decided to keep this organization api in LMS for now: https://github.com/edx/edx-platform/pull/11096